### PR TITLE
Create home page for new docs site

### PR DESCRIPTION
**Pull request by Ray2126**
Test plan:

- Go to docs.centrapay.com/v2 and see that it looks like 
image

### CODE REVIEW

**Review by tom-blake — APPROVED** [review]

**Review by AndyClifford — APPROVED** [review]

File: new-docs/components/Card.vue [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<template>
+  <div
```
Comment (MeganSteenkamp) [inline new-docs/components/Card.vue:2]: @Ray2126 I know this is merged but some feedback would be that this should be an anchor tag 👍

Comment (AndyClifford) [reply new-docs/components/Card.vue:2]: Would this avoid needing an onClick method?

Comment (MeganSteenkamp) [reply new-docs/components/Card.vue:2]: @AndyClifford yes

Comment (Ray2126) [reply new-docs/components/Card.vue:2]: Thanks @MeganSteenkamp , I have a PR up to replace the div with a `NuxtLink` which is Nuxt's version of an anchor tag.

Will change the `h6` and `header` tags as well :)

**Pull request merged**